util/util.go

```diff
@@ -1,9 +1,37 @@
 package util
 
-import "os/exec"
+import (
+	"errors"
+	"net"
+	"os/exec"
+	"strings"
+)
 
 // IsCommandAvailable check command exits.
 func IsCommandAvailable(cmd string) bool {
 	_, err := exec.LookPath(cmd)
 	return err == nil
 }
+
+func GetClientIp() (string, error) {
+	addresses, err := net.InterfaceAddrs()
+
+	if err != nil {
+		return "", err
+	}
+
+	for _, address := range addresses {
+		if ipnet, ok := address.(*net.IPNet); ok && !ipnet.IP.IsLoopback() {
+			if ipnet.IP.To4() != nil {
+				return ipnet.IP.String(), nil
+			}
+
+		}
+	}
+
+	return "", errors.New("can not find the client ip addresses")
+}
+
+func CountWords(s string) int {
+	return len(strings.Fields(s))
+}
```

Comment:
Overall, the code patch seems to be fine and doesn't contain any major bug risks or security vulnerabilities. However, here are some suggestions for improvement:

1. Error Handling: The code doesn't have proper error handling. For example, in the `IsCommandAvailable` function, if `exec.LookPath` returns an error, it's just ignored. It would be better to return the error or log it.

2. Naming Convention: The function name `IsCommandAvailable` could be improved to be more descriptive. A better name could be `IsExecutableAvailable` or something similar.

3. Code Duplication: The `import` statement for `os/exec` is duplicated. It would be better to remove the duplicate import.

4. Code Formatting: The code could be formatted better to improve readability. For example, there are unnecessary empty lines and inconsistent use of braces.

5. Testing: There are no tests included with the code patch. It's always a good practice to include tests to ensure that the code works as expected and to catch any regressions.